### PR TITLE
fix plugin-types-usage.md

### PR DESCRIPTION
**Pull request by sympatheticmoose**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docusaurus/docs/introduction/plugin-types-usage.md [hunk 1]
```diff
@@ -77,13 +77,13 @@ Not all plugins support this type of configuration. Refer to the readme of speci
...
+Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) so that you can define Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
 
 Note that a plugin of type `datasource` must be installed before it can be provisioned.
 
 ### Bundling of dashboards
 
-Data-source plugins can [include dashboards]((https://grafana.com/developers/plugin-tools/reference-plugin-json#includes)) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
+Data-source plugins can [include dashboards](https://grafana.com/developers/plugin-tools/reference-plugin-json#includes) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
```
Comment (jackw) [inline docusaurus/docs/introduction/plugin-types-usage.md:86]: We shouldn't navigate from plugin-tools docs to the live site.  I believe we can shorten these to `/<path-to-page>`.

```suggestion
Data-source plugins can [include dashboards](/reference-plugin-json#includes) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
```

File: docusaurus/docs/introduction/plugin-types-usage.md [hunk 2]
```diff
@@ -117,8 +117,8 @@ Note that the plugin must be installed before provisioning can succeed with a `p
 
 ### Bundling of apps
 
-The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or [include]((https://grafana.com/developers/plugin-tools/reference-plugin-json#includes)) multiple plugins in the same package. 
+The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or [include](https://grafana.com/developers/plugin-tools/reference-plugin-json#includes) multiple plugins in the same package.
```
Comment (jackw) [inline docusaurus/docs/introduction/plugin-types-usage.md:120]:
```suggestion
The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or [include](/reference-plugin-json#includes) multiple plugins in the same package.
```

**Review by jackw — CHANGES_REQUESTED** [review]

**Comment by sympatheticmoose** [comment]
good point - didn't catch ☕

**Review by jackw — APPROVED** [review]
Awesome stuff! 🚀

**Pull request merged**